pkgs/spark-wallet/generate.sh

```diff
@@ -1,3 +1,26 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p nodePackages.node2nix
-exec node2nix --nodejs-10 -i pkg.json -c composition.nix --no-copy-node-env --supplement-input supplement.json
+#! nix-shell -i bash -p nodePackages.node2nix gnupg wget jq moreutils
+set -euo pipefail
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+
+# Get/verify spark-wallet-npm.tgz
+version=$(cat version)
+export GNUPGHOME=$TMPDIR
+gpg --keyserver hkps://hkps.pool.sks-keyservers.net --recv-key FCF19B67866562F08A43AAD681F6104CD0F150FC
+wget -P $TMPDIR https://github.com/shesek/spark-wallet/releases/download/v${version}/SHA256SUMS.asc
+wget -P $TMPDIR https://github.com/shesek/spark-wallet/releases/download/v${version}/spark-wallet-${version}-npm.tgz
+(cd $TMPDIR; gpg --verify $TMPDIR/SHA256SUMS.asc; sha256sum -c --ignore-missing $TMPDIR/SHA256SUMS.asc)
+shasum=$(sha256sum $TMPDIR/spark-wallet-${version}-npm.tgz | cut -d\  -f1)
+
+# Make qrcode-terminal dependency
```

Comment:
Please add a comment why this is needed.

Reply:
Resolved in e1527c7

Reply:
Sorry, I meant that it wasn't clear to me why the hacking packing.json hacking was needed in the first place.
But I think I get it now.
Here's a comment that would clear it up:
`Make qrcode-terminal a strict dependency so that node2nix includes it in the package derivation.`

Reply:
Resolved in 9049f8d

I get the confusion. I would love to fix it upstream in node2nix but I couldn't do it when I tried.